GeoLib/Grid.h

```diff
@@ -201,6 +201,10 @@ Grid<POINT>::Grid(InputIterator first, InputIterator last,
                                     _max_pnt[1] - _min_pnt[1],
                                     _max_pnt[2] - _min_pnt[2]}};
 
+    // enlarge delta
+    for (auto & d : delta)
+        d = std::nextafter(d, std::numeric_limits<double>::max());
```

Comment:
`delta` is the space diagonal of the bbox of the grid? Why do you have to extend it such a tiny bit?